packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactNode } from 'react';
+import { LinkProps, packs } from '../core';
+import { TextLink, TextLinkExternal } from '../text-link';
+import { Box } from '../box';
+import { Flex } from '../flex';
+import { Stack } from '../stack';
+import { Text } from '../text';
+import { ArrowRightIcon } from '../icon';
+import { FeatureLinkListBackground, hoverColorMap } from './utils';
+
+export type FeatureLinkListItemProps = LinkProps & {
+	label: ReactNode;
+	description: ReactNode;
+	background?: FeatureLinkListBackground;
+};
+
+export const FeatureLinkListItem = ({
+	label,
+	description,
+	background = 'body',
+	...props
+}: FeatureLinkListItemProps) => {
+	const TextComponent = props.target == '_blank' ? TextLinkExternal : TextLink;
+
+	return (
+		<Box
+			as="li"
+			borderBottom
+			paddingBottom={0.25}
+			paddingTop={0.25}
+			css={{
+				'&:hover': {
+					backgroundColor: hoverColorMap[background],
+				},
+			}}
+		>
+			<Flex
+				gap={1.5}
+				paddingY={1}
```

Comment:
According to the desigm this should be `1.5`. Doing this would allow you to remove the parent box and move those props to the Flex
<img width="838" alt="image" src="https://github.com/user-attachments/assets/72bb66d6-f975-4343-8354-db6b7cba3daf">

Reply:
Hmm removing outer box would mean the focus indicator would get cropped, so perhaps just change the `Box` to `paddingY={0.5}` 
<img width="712" alt="image" src="https://github.com/user-attachments/assets/28bdd0b0-a2b9-424e-ba34-a8696fdce9fa">